capstone/target/appengine-staging/__static__/flags_script.js

```diff
@@ -0,0 +1,204 @@
+    data: getPoints(),
+    map: map
+  });
+  //map.clearOverlays(); //clear markers
+  let card = document.getElementById('pac-card');
+  let input = document.getElementById('pac-input');
+  let types = document.getElementById('type-selector');
+  let strictBounds = document.getElementById('strict-bounds-selector');
+  let autocomplete = new google.maps.places.Autocomplete(input);
+  map.controls[google.maps.ControlPosition.TOP_RIGHT].push(card);
+ 
+  // Bind the map's bounds (viewport) property to the autocomplete object,
+  // so that the autocomplete requests use the current map bounds for the
+  // bounds option in the request.
+  autocomplete.bindTo('bounds', map);
+ 
+  // Set the data fields to return when the user selects a place.
+  autocomplete.setFields(
+  ['address_components', 'geometry', 'icon', 'name']);
+ 
+  let infowindow = new google.maps.InfoWindow();
+  let infowindowContent = document.getElementById('infowindow-content');
+  infowindow.setContent(infowindowContent);
+  let marker = new google.maps.Marker({
+    map: map,
+    anchorPoint: new google.maps.Point(0, -29),
+  });
+  autocomplete.addListener('place_changed', function() {
+    infowindow.close();
+    marker.setVisible(false);
+    let place = autocomplete.getPlace();
+    if (!place.geometry) {
+      // User entered the name of a Place that was not suggested and
+      // pressed the Enter key, or the Place Details request failed
+      window.alert('No details available for input: \'' + place.name + '\'');
+      return;
+    }
+ 
+    // If the place has a geometry, then present it on a map
+    if (place.geometry.viewport) {
+      map.fitBounds(place.geometry.viewport);
+    } else {
+      map.setCenter(place.geometry.location);
+      map.setZoom(17);  // Why 17? Because it looks good.
+    }
+    
+    marker.setPosition(place.geometry.location);
+    //if place is near the user, let them submit
+    marker.setVisible(true);
+ 
+    let address = '';
+    
+    if (place.address_components) {
+      address = [
+        (place.address_components[0] && place.address_components[0].short_name || ''),
+        (place.address_components[1] && place.address_components[1].short_name || ''),
+        (place.address_components[2] && place.address_components[2].short_name || '')
+      ].join(' ');
+    }
+ 
+    infowindowContent.children['place-icon'].src = place.icon;
+    infowindowContent.children['place-name'].textContent = place.name;
+    infowindowContent.children['place-address'].textContent = address;
+    infowindowContent.children['location'].textContent = place.geometry.location;
+    localStorage.setItem("form-place-name", place.name);
+    localStorage.setItem("form-place-address", address);
+    localStorage.setItem("form-lat", place.geometry.location.lat());
+    localStorage.setItem("form-long", place.geometry.location.lng());
+
+    /*document.getElementById('form-place-name').textContent = place.name;
```

Comment:
remove commented out code, here and other places in these capstone/target/appengine-staging/__static__/ files